supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/api/RunStatus.scala

```diff
@@ -28,8 +28,8 @@ object RunStatus {
   }
 
   sealed trait UnsuccessfulRunStatus extends TerminalRunStatus {
-    val errorMessage: Option[String]
-    lazy val prettyPrintedError: String = errorMessage map { e => s" Message: $e" } getOrElse ""
+    val errorMessages: List[String]
```

Comment:
ToL Can it be empty?  If not then `NonEmptyList` perhaps?

Reply:
I think technically it could, if for some reason the PAPI response didn't have an error message or we could not extract it properly.